test/apache_properties/schema-registry.properties

```diff
@@ -25,7 +25,7 @@ listeners=http://0.0.0.0:8081
 # (see zookeeper docs for details).
 # This is a comma separated host:port pairs, each corresponding to a zk
 # server. e.g. "127.0.0.1:3000,127.0.0.1:3001,127.0.0.1:3002".
-kafkastore.connection.url=localhost:2181
+#kafkastore.connection.url=localhost:2181
```

Comment:
could you explain the change in this file?

Reply:
I ran into schema registry issues while running version 7.2.1 in confluent. the fix was to not use kafkastore.